tests/lang/s02_core/s01_user_defined_data_types/t03_inheritance.nim

```diff
@@ -52,8 +52,9 @@ block derive_from_ref_root_obj:
   ## To convert base type back to derived, you can use object conversion
   doAssert Derived(base).f2 == 12
 
-  when not defined(js):
+  when not defined(js) and not defined(vm):
    when not defined(js) or defined(tryBrokenSpecification):
```

Comment:
@saem The outer `when` seems redundant?

Reply:
yeah, that was a hasty add, pretty sure that can be consolidated.